src/ConversionContext.h

```diff
@@ -34,7 +38,10 @@ namespace drafter
             return registry;
         }
 
-        ConversionContext(const WrapperOptions& options) : options(options) {}
+        inline const NewLinesIndex& GetNewLinesIndex() const
```

Comment:
Why `inline`? Fun fact: `inline` causes GCC to produce a weak symbol, that's all.